dashboard/test/controllers/omniauth_callbacks_controller_test.rb

```diff
@@ -249,4 +249,79 @@ class OmniauthCallbacksControllerTest < ActionController::TestCase
     user = User.last
     assert_equal auth[:credentials][:token], user.oauth_token
   end
+
+  # The following tests actually test the user model, but relate specifically to
+  # oauth uniqueness checks so they are included here. These have not been working
+  # in the past for subtle reasons.
+
+  test "omniauth student is checked for email uniqueness against student" do
+    email = 'duplicate@email.com'
+    original_user = create(:user, email: email)
+    assert_empty original_user.email
```

Comment:
Is this assert needed here? This empty student email functionality is already [tested](https://github.com/code-dot-org/code-dot-org/blob/d03519d1a4977b5f7c6abfb5086d25edfef519dd/dashboard/test/models/user_test.rb#L641-L644) in the user model.

Reply:
Not particularly, I was mostly just satisfying myself that I was testing what I thought I was testing. Happy to remove.